integration/images/volume-ownership/tools/get_owner_windows.go

```diff
@@ -20,13 +20,14 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"path/filepath"
 
 	"golang.org/x/sys/windows"
 )
 
 func main() {
 	if len(os.Args) != 2 {
-		fmt.Printf("Usage: %s file_or_directory\n", os.Args[0])
+		fmt.Printf("Usage: %s file_or_directory\n", filepath.Base(os.Args[0]))
```

Comment:
I didn't know that POSIX has recommendations around such a detail...

This one seems only used by `get_owner_windows.exe`. Should we use the name instead?

Reply:
> I didn't know that POSIX has recommendations around such a detail...

Yes, it's quite detailed, but also looks like nobody _really_ follows the recommendations (including quite "posix-y" tools). That said; I thought it made sense to not make the output depend on "how" you call the binary (e.g. don't print `../../../../<binary>` if you happen to use that as path 😂)

There were a couple more (somewhat interesting) recommendations in there, for example: https://www.gnu.org/prep/standards/html_node/_002d_002dversion.html#g_t_002d_002dversion

> the version number proper starts after the last space

(could be useful for easily parse / get the version from the output, especially given that we also output "git commit" etc.

Some other recommendations about licensing, and "support" / "project" URLs in output of `--help` (perhaps URL to where to report security issues etc?) https://www.gnu.org/prep/standards/html_node/_002d_002dhelp.html#g_t_002d_002dhelp

Anyway; not very urgent, but more of a "nice to do at some point"

> This one seems only used by get_owner_windows.exe. Should we use the name instead?

oh, good on; I guess for this one, we could hard code it; let me change that

Reply:
Well, I will dust off the rock I crawl under when I forget guidelines I should never forget. Thanks for taking care of this!

Reply:
You're welcome 😅. Yeah (as outlined above); even though there's POSIX, it looks like nobody really follows the rules, so I consider it a "best effort" (and something to keep in the back of my mind when working on these things)